setup.py

```diff
@@ -61,6 +64,12 @@
     )
     ext_modules.append(ext)
 
+# Hack to install ta-lib library loading into virtualenv
+def install_virtualenv_lib_loader():
+    with open(os.getenv('VIRTUAL_ENV', './venv') + '/bin/activate', 'a') as f:
```

Comment:
I assume this should check for exists first?

```python
activate = os.getenv('VIRTUAL_ENV', './venv') + '/bin/activate'
if os.path.exists(activate):
    with open(activate) as f:
        f.write("\nexport LD_LIBRARY_PATH=$VIRTUAL_ENV/lib:$LD_LIBRARY_PATH\n")
```

I have not used virtual env much, but is ``$VIRTUAL_ENV/lib`` a standard location for loading libraries, or a workaround for ``talib``?  Also, is ``$VIRTUAL_ENV/bin/activate`` a standard script that is run?